packages/ui-components/src/components/action-button/usage/javascript.md

```diff
@@ -0,0 +1,13 @@
+```html
```

Comment:
@ricardogoncalves89 should we remove the JS and Stencil usage examples for consistency across documentation?

Reply:
I think so